client/src/shared/components/Card/index.tsx

```diff
@@ -0,0 +1,29 @@
+interface CardProps {
+  height: string
+  title: string
+  content: string
+}
+
+const Card: React.FC<CardProps> = ({ height, title, content }: CardProps) => {
+  return (
+    // allows the card component to be responsive based on their screensize
+    <div className="w-full md:w-1/2 lg:w-1/3">
+      <div
+        className={`bg-white rounded-lg overflow-hidden shadow-lg ${height} w-auto`}
+      >
+        <div className="px-6 py-4">
+          <h3 className="text-lg font-medium text-gray-900 mb-2">{title}</h3>
+          <p className="text-gray-600">{content}</p>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+Card.defaultProps = {
+  height: 'h-30',
+  title: 'Card Title',
+  content: 'Lorem Ipsum...'
```

Comment:
- do not set any specific default values in the props. Please set it as empty.
 ```suggestion
        className={`bg-white rounded-lg overflow-hidden shadow-lg h-30 ${height} w-auto`}
      >
        <div className="px-6 py-4">
          <h3 className="text-lg font-medium text-gray-900 mb-2">{title}</h3>
          <p className="text-gray-600">{content}</p>
        </div>
      </div>
    </div>
  );
};

Card.defaultProps = {
  height: '',
  title: '',
  children: React.ReactNode
```